packages/gestalt/dist/gestalt.js.flow

```diff
@@ -1,2 +1,2 @@
 // @flow
-export * from '../src';
+export * from './src.js';
```

Comment:
Is this one actually a file called `src.js`? I thought this reaches into the `src` folder hence the `..`